test/models/populations.js

```diff
@@ -0,0 +1,21 @@
+const assert = require('assert');
+const _ = require('lodash');
+const fs = require('fs-extra');
+const path = require('path');
+const { v4: uuid } = require('uuid');
+
+const helper = require('../helper');
+const models = require('../../models');
+
+describe('models.SitePopulation', () => {
+  it('creates a new record', async () => {
+    let record = models.Population.build({
+      name: 'youngsters',
+    });
+    assert.deepStrictEqual(record.id, null);
+    await record.save();
+    assert(record.id);
+    test = await models.Population.findByPk(record.id);
+    console.log(test);
```

Comment:
Instead of a `console.log`, which adds clutter to the output, how about another assertion? Like making sure that the name is what you expect it to be?